[BUG] Faq page is broken

### Description

faq page has bugs and is broken

- wait compiling /favicon.ico/route (client and server)...
- error ./node_modules/lucide-react/dist/cjs/lucide-react.js:2:0
Module not found: Can't resolve 'D:\Open Source\info\informatician\node_modules\lucide-react\dist\cjs\icons\minus-circle'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./app/faq/Accordion.jsx
- warn Fast Refresh had to perform a full reload. Read more: https://nextjs.org/docs/messages/fast-refresh-reload
- wait compiling...
- error ./app/faq/Accordion.jsx:4:0
Module not found: Can't resolve 'lucide-react/icons/minus-circle'
  2 |
  3 | import { PlusCircle } from "lucide-react";
> 4 | import MinusCircleIcon from 'lucide-react/icons/minus-circle';
  5 |
  6 | import { useState } from "react";
  7 |

https://nextjs.org/docs/messages/module-not-found
- warn Fast Refresh had to perform a full reload. Read more: https://nextjs.org/docs/messages/fast-refresh-reload


### Screenshots

![image](https://github.com/rohansx/informatician/assets/96341431/baec7ee1-1b13-45ba-b24f-46a55651b34a)


### Any additional information?

_No response_

### What browser are you seeing the problem on?

_No response_

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rohansx/informatician/blob/main/CONTRIBUTING.md)
- [X] I have read the discussion tab thoroughly and got the project idea
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

--- a/app/faq/Accordion.jsx
+++ b/app/faq/Accordion.jsx
@@ -1,6 +1,6 @@
 "use client";
 
-import { MinusCircle, PlusCircle } from "lucide-react";
+// use client
 import { useState } from "react";
 
 export default function AccordionItem({ title, content }) {
@@ -27,11 +27,11 @@ export default function AccordionItem({ title, content }) {
             expanded ? "rotate-90" : "rotate-0"
           }`}
         >
-          {expanded ? <PlusCircle size={18} /> : <MinusCircle size={18} />}
+          {expanded ? "-" : "+"}
         </span>
       </button>
       <div
-        className={`mt-3 overflow-hidden  duration-300 ${
+        className={`mt-3 overflow-hidden duration-300 ${
           expanded ? "max-h-60" : "max-h-0"
         }`}
       >

